ReactAndroid/src/main/java/com/facebook/react/modules/statusbar/StatusBarModule.java

```diff
@@ -50,17 +53,33 @@ public String getName() {
     return "StatusBarManager";
   }
 
-  @Override
-  public @Nullable Map<String, Object> getConstants() {
+  // TODO(dantman): Detect rare status bar height changing events (such as a Phone+Tablet or docking device that can change
+  // screen dimensions at runtime) and emit a statusBarFrameDidChange when they happen.
+
+  private float getStatusBarHeight() {
     final Context context = getReactApplicationContext();
     final int heightResId = context.getResources()
       .getIdentifier("status_bar_height", "dimen", "android");
-    final float height = heightResId > 0 ?
+    return heightResId > 0 ?
       PixelUtil.toDIPFromPixel(context.getResources().getDimensionPixelSize(heightResId)) :
       0;
+  }
 
+  @Override
```

Comment:
Where do we use this?
Exporting constants from Module turns off some "lazy" performance optimization, so if we don't really need this, we have to remove it.

Reply:
This is the source of the `StatusBar.currentHeight` property. All I've done for Android is move most of the code in `getConstants` to a common method that is used by both `getConstants` so the old `currentHeight` still works till we drop it. And the new cross-platform `getHeight`.